compute/cloud-client/src/main/java/compute/preemptible/CreatePreemptibleInstance.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package compute.preemptible;
+
+// [START compute_preemptible_create]
+
+import com.google.cloud.compute.v1.AttachedDisk;
+import com.google.cloud.compute.v1.AttachedDiskInitializeParams;
+import com.google.cloud.compute.v1.InsertInstanceRequest;
+import com.google.cloud.compute.v1.Instance;
+import com.google.cloud.compute.v1.InstancesClient;
+import com.google.cloud.compute.v1.NetworkInterface;
+import com.google.cloud.compute.v1.Operation;
+import com.google.cloud.compute.v1.Scheduling;
+import java.io.IOException;
+import java.util.concurrent.ExecutionException;
+
+public class CreatePreemptibleInstance {
+
+  public static void main(String[] args)
+      throws IOException, ExecutionException, InterruptedException {
+    // TODO(developer): Replace these variables before running the sample.
+    // projectId: project ID or project number of the Cloud project you want to use.
+    // zone: name of the zone you want to use. For example: “us-west3-b”
+    // instanceName: name of the new virtual machine.
+    String projectId = "your-project-id-or-number";
+    String zone = "zone-name";
+    String instanceName = "instance-name";
+
+    createPremptibleInstance(projectId, zone, instanceName);
+  }
+
+  // Send an instance creation request with preemptible settings to the Compute Engine API
+  // and wait for it to complete.
+  public static void createPremptibleInstance(String projectId, String zone, String instanceName)
+      throws IOException, ExecutionException, InterruptedException {
+
+    String machineType = String.format("zones/%s/machineTypes/e2-small", zone);
+    String sourceImage = String
+        .format("projects/debian-cloud/global/images/family/%s", "debian-11");
```

Comment:
is `debian-11` a constant? If so, why not just inline the entire string? If not, can we make it a variable?

Reply:
It's a variable. Since, the other samples also follow this particular way of initialization, I've stuck to the same.

Reply:
Done. Thanks.